src/tmx/TmxApi/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.10
+
+# Relative path conversion top directories.
+set(CMAKE_RELATIVE_PATH_TOP_SOURCE "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx")
+set(CMAKE_RELATIVE_PATH_TOP_BINARY "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx")
```

Comment:
This file contains specific username home path. Can we remove this cache file? or git ignore it?